firmware/controllers/map_averaging.cpp

```diff
@@ -170,6 +178,16 @@ static void endAveraging(void *arg) {
 #if EFI_PROD_CODE || defined(__DOXYGEN__)
 	v_averagedMapValue = adcToVoltsDivided(
 			mapAccumulator / mapMeasurementsCounter);
+	// todo: move out of locked context?
+	averagedMapRunningBuffer[averagedMapBufIdx] = getMapByVoltage(v_averagedMapValue);
+	// increment circular running buffer index
+	averagedMapBufIdx = (averagedMapBufIdx + 1) % mapMinBufferLength;
+	// find min. value (only works for pressure values, not raw voltages!)
+	minMapPressure = averagedMapRunningBuffer[0];
```

Comment:
Do not we need to run this loop with a local variable? otherwise would not we have a chance to read the un-minimalized value once in a while as long as we read not under a lock?

we probably never read not under a lock in reality so we can either extract a local variable or add a comment clarifying that we are aware of this concurn but choose not to care since we believe we should not care.